drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/utilities/ColumnUtilitiesGetValueListTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
```

Comment:
I am doing this so long time, that maybe I started previous year. Just kidding :) I will do.